BSP/Inc/BSP_GPIO.h

```diff
@@ -16,10 +16,10 @@ typedef enum {PORTA = 0, PORTB, PORTC, PORTD, NUM_PORTS} port_t;
 
 /**
  * @brief   Initializes a GPIO port
- * @param   port to initialize
+ * @param   port -port to initialize, mask - pins, write - input or output
```

Comment:
Just a documentation thing. You can use multiple `@param` lines for multiple parameters. Just like how the other methods in here have done. Just in case we ever do some automatic documentation generator, it'll want it to be in that format.